### PR TITLE
add footer

### PR DESCRIPTION
**Pull request by bellesea**

### CODE REVIEW

**Review by eilla1 — APPROVED** [review]
checked it out & lgtm!

File: config.toml [hunk 1]
```diff
@@ -282,6 +282,38 @@ disableHugoGeneratorInject = false
...
+    name = "Advisors"
+    url  = "https://foundation.dogecoin.com/about/"
+
+  [[params.footerFoundation]]
+    name = "Blog"
+    url  = "https://foundation.dogecoin.com/announcements/"
+
+  [[params.footerFoundation]]
+    name = "Roadmap"
+    url  = "https://foundation.dogecoin.com/roadmap/v"
```
Comment (eilla1) [inline config.toml:319]: i think the link is just "https://foundation.dogecoin.com/roadmap/" without the /v at the end

Comment (bellesea) [reply config.toml]: yes, you're right! thanks ella! just made the changes :))

**Comment by quackduck** [comment]
you guys are so awesome

File: themes/hello-friend-ng/layouts/partials/footer.html [hunk 2]
```diff
@@ -1,19 +1,56 @@
...
+    <div class="footer">
+        <div class="footer__inner">
+            <div class="footer__content">
+                <div class="footer__content-left">
+                <div>
+                    © 2014-{{ now.Year }} | The Dogecoin Project
+                </div>
+                </div>
+                <div>
+                    Website Design - CE | <span>Coding - Florian & xanimo</span>
```
Comment (quackduck) [inline themes/hello-friend-ng/layouts/partials/footer.html:51]: :eyes:

```suggestion
                    Website Design - CE | <span>Coding - Florian, xanimo, Belle and Ella</span>

```

Comment (eilla1) [reply themes/hello-friend-ng/layouts/partials/footer.html:51]: I actually haven't contributed a feature to this site yet. _school_ 😭 
Workin' on it though!

Comment (quackduck) [reply themes/hello-friend-ng/layouts/partials/footer.html:51]: I see you guys making PRs all the time tho!

**Review by xanimo — CHANGES_REQUESTED** [review]
This looks good! Thank you and sorry for the delay. Do you know how to rebase and squash your commits into one @bellesea? DM on discord me if not. If I could request that you do that, would be only ask before merging.

**Comment by xanimo** [comment]
> This looks good! Thank you and sorry for the delay. Do you know how to rebase and squash your commits into one @bellesea? DM on discord me if not. If I could request that you do that, would be only ask before merging.

And bonus points if you want to add `overflow-y: hidden;` to L31 in _main.scss as I noticed overflow under the footer on home page. Don't worry if not, it would be a simple PR to open for someone.

**Comment by bellesea** [comment]
> > This looks good! Thank you and sorry for the delay. Do you know how to rebase and squash your commits into one @bellesea? DM on discord me if not. If I could request that you do that, would be only ask before merging.
> 
> And bonus points if you want to add `overflow-y: hidden;` to L31 in _main.scss as I noticed overflow under the footer on home page. Don't worry if not, it would be a simple PR to open for someone.

No worries! I didn't but just learned haha ( hopefully i did it correctly? ). Also, the overflow was due to negative bottom with absolute positioning on the map image so i removed it -- i think it fixed?

Thank you!

File: themes/hello-friend-ng/assets/scss/_main.scss [hunk 3]
```diff
@@ -26,8 +26,8 @@ body {
   -webkit-text-size-adjust: 100%;
 
   display: flex;
-  min-height: 100vh;
   flex-direction: column;
+  height: 100%;
```
Comment (xanimo) [inline themes/hello-friend-ng/assets/scss/_main.scss:31]: I'm sorry to request changes again but can you add that `min-height: 100vh;` again? Without it it doesn't align with the bottom of the viewport:
Screenshot from 2021-11-22 13-07-39

**Review by xanimo — CHANGES_REQUESTED** [review]
Everything looks great with exception to the removal of that `min-height: 100vh`. If you can add that back will ACK.

And great job with the rebase + squash! Was exactly what was required! :)

**Review by xanimo — APPROVED** [review]

**Comment by xanimo** [comment]
ACK. Am going to merge and then open a new PR with the previous change requested.

**Comment by bellesea** [comment]
> ACK. Am going to merge and then open a new PR with the previous change requested.

hey @xanimo, i'm sorry about the really late response to your request! I'll make the PR. Apologies 😓.

**Comment by xanimo** [comment]
> > ACK. Am going to merge and then open a new PR with the previous change requested.
> 
> hey @xanimo, i'm sorry about the really late response to your request! I'll make the PR. Apologies 😓. 

Hey Bella no it's all good don't worry about! ♥️ Thank so much for all your hard work!

**Pull request merged**